stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/ui/Overview.java

```diff
@@ -1,27 +1,33 @@
 package io.getstream.chat.docs.java.ui;
 
-import androidx.appcompat.app.AppCompatActivity;
-import androidx.lifecycle.LifecycleOwner;
+import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProvider;
 
 import io.getstream.chat.android.ui.feature.messages.list.MessageListView;
+import io.getstream.chat.android.ui.viewmodel.messages.MessageListViewModel;
 import io.getstream.chat.android.ui.viewmodel.messages.MessageListViewModelBinding;
 import io.getstream.chat.android.ui.viewmodel.messages.MessageListViewModelFactory;
-import io.getstream.chat.android.ui.viewmodel.messages.MessageListViewModel;
 
 /**
  * [Overview](https://getstream.io/chat/docs/sdk/android/ui/overview/)
  */
-public class Overview extends AppCompatActivity {
+public class Overview extends Fragment {
 
-    public void viewModelsInitialization(MessageListView messageListView, LifecycleOwner viewLifecycleOwner) {
-        ViewModelProvider.Factory factory = new MessageListViewModelFactory // 1
+    private MessageListView messageListView;
+
+    /**
+     * [ViewModels](https://getstream.io/chat/docs/sdk/android/ui/overview/#viewmodels)
+     */
+    public void viewModels() {
+        // 1
+        ViewModelProvider.Factory factory = new MessageListViewModelFactory
                 .Builder()
-                .cid("channelType:channelId")
```

Comment:
I think I actually prefer `channelType:channelId` as it is more explicit.
What do you think?

Reply:
I'm leaving `messaging:123` as it's slightly more common in our codebase.